ADCtoAmpConverter.c

```diff
@@ -0,0 +1,68 @@
+#include "ADCtoAmpConverter.h"
+#include<stdio.h>
+
+#define CONVERT_ADC_AMP(AdcValue)   ((float)((AdcParameterStruct.AdcConversionScale*(ADCValue))/(float)(AdcParameterStruct.MaxValueReadByAdc)) + (float)AdcParameterStruct.AdcConversionOffset)
+#define IS_GREATER_THAN_MAX_VALUE(ADCValue)  (ADCValue > AdcParameterStruct.MaxValueReadByAdc)
+
+
+static int RoundOffTo_NearInteger(float num)
+{
+    int tempCheck, roundNum, tempNum;
+
+    if(num>0)
+    {
+        tempNum = num*10;
+        tempCheck = tempNum%10;
+        if(tempCheck>=5)
+        {
+            roundNum = num;
+            roundNum++;
+        }
+        else
+        {
+            roundNum = num;
+        }
+     return roundNum;
+        printf("\nWhole number after rounding off the given real number = %d", roundNum);
+    }
+    else if(num<0)
+    {
+        num = -(num);
```

Comment:
please check if the code behaves as expected with the value = -5.6, as far as i can see, it returns a value of 4 (which is a >0 value and not right)which is incorrect. Please check...